 Widget: Catch null point exception in AbsListViewAutoScroller

The first view of AbsListView is null, so there is a NullPointerException
happen.

Add a judgement for the first view of AbsListView.

diff --git a/core/java/com/android/internal/widget/AutoScrollHelper.java b/core/java/com/android/internal/widget/AutoScrollHelper.java
index 66fa002..0114644 100644
--- a/core/java/com/android/internal/widget/AutoScrollHelper.java
+++ b/core/java/com/android/internal/widget/AutoScrollHelper.java
@@ -912,7 +912,7 @@ public boolean canTargetScrollVertically(int direction) {
                 // Are we already showing the entire first item?
                 if (firstPosition <= 0) {
                     final View firstView = target.getChildAt(0);
-                    if (firstView.getTop() >= 0) {
+                    if (firstView == null || firstView.getTop() >= 0) {
                         return false;
                     }
                 }
